src/Nancy/Json/Simple/NancySerializationStrategy.cs

```diff
@@ -0,0 +1,217 @@
+﻿namespace Nancy.Json.Simple
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using Nancy.Extensions;
+
+    internal class NancySerializationStrategy : PocoJsonSerializerStrategy

```

Comment:
problem with this is SimpleJson is marked as internal via an ifdef, we can change that to public if needs be but as it stands there would be an issue as this would be public but its inherited type would be internal. What should be done?